modules/CmpdRegBulkLoader/src/server/routes/CmpdRegBulkLoaderRoutes.coffee

```diff
@@ -338,3 +343,103 @@ exports.purgeFile = (req, resp) ->
 				console.log response
 				resp.end JSON.stringify "Error"
 		)
+
+checkStatus = (response) ->
+	# check for http error
+	if response.ok
+		return response
+	else
+		throw new HTTPResponseError response
+
+class HTTPResponseError extends Error
+	constructor: (response, ...args) ->
+		super("HTTP Error Response: #{response.status} #{response.statusText}", ...args)
+		this.response = response
+
+exports.getSDFFromBulkFileId = (req, resp) ->
+	req.setTimeout 86400000
+	config = require '../conf/compiled/conf.js'
+	baseurl = config.all.client.service.cmpdReg.persistence.fullpath+"bulkload/getLotsByBulkLoadFileID?bulkLoadFileID=" +req.params.bulkFileID
+	request = require 'request'
+	request(
+		method: 'GET'
+		url: baseurl
+		json: true
+	, (error, response, lotCorpNames) =>
+		if !error && response.statusCode == 200
+			#make a second call using the returned lots to get an SDF file
+			baseurl = config.all.client.service.cmpdReg.persistence.fullpath+"export/lotCorpNames"
+			try
+			# Use fetch to call the service and pipe the response to the client
+				response = await fetch(baseurl,
+					method: 'POST'
+					body: JSON.stringify(lotCorpNames)
+					headers:
+						'Content-Type': 'application/json'
+				)
+				checkStatus response
+				resp.set({
+					"content-length": response.headers.get('content-length'),
+					"content-type": response.headers.get('content-type'),
+				})
+				response.body.pipe(resp);
+			catch err
+				console.log "Error calling service: " + err
+				resp.statusCode = 500
+				resp.json err
+
+		else
+			console.log 'got ajax error trying to retrieve lots from bulk file ID'
+			console.log error
+			console.log json
+			console.log response
+			resp.end JSON.stringify "Error"
```

Comment:
Do we want to pass through any more information to the response besides "Error' is console logging acceptable on its own